ccontainermain.go

```diff
@@ -269,7 +271,7 @@ func shutdownCaché(inst string) (bool, error) {
 	}
 
 	// check that the shutdown was successful
-	if err := checkCmdOutcome("down", inst); err != nil {
+	if err := checkCmdOutcome("down", inst, false); err != nil {
 		log.Printf("Error: Caché was not shutdown successfully.\n")
```

Comment:
OK Thanks @daimor. If i remember correctly I already parse the string for the relevant message. Either send the whole string before parsing or the parsed part... you'll have to change functions signature... Thanks